### ISSUE TITLE
Add New Customer Model to Migration Tool and Populate Migrated Database Table

### ISSUE THREAD

**Issue by aidandunsdon** [openfoodfoundation/openfoodnetwork#383]

**Comment by aidandunsdon** [comment]
@lin-d-hop @pmackay @NickWeir63 

Hi Lynne  I have run a git pull origin a few times and a rake:dbmigrate since the team meeting where we identified that I needed to populate a new customer entity with customer ids from the Stroudco database. I was expecting to find a rails migration to create a new customer table but it doesn't appear to be merged with the most recent git pull origin.

I notice that the commit for New customer model #329 at https://github.com/openfoodfoundation/openfoodnetwork/commit/7bcc4bfa858d32ab3271b26c5e4d3098ebc8e23d called LD adding new customer model seems to be failing with error message "Travis CI Build failed" Not sure what this means or the current status of this merge. Am I missing something here - I seem to be blocked on this task #383
